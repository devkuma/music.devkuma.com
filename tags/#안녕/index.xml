<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Music – /#안녕</title>
    <link>https://music.devkuma.com/tags/#%EC%95%88%EB%85%95/</link>
    <image>
      <url>https://music.devkuma.com/tags/#%EC%95%88%EB%85%95/logo/180x180.jpg</url>
      <title>/#안녕</title>
      <link>https://music.devkuma.com/tags/#%EC%95%88%EB%85%95/</link>
    </image>
    <description>Recent content in /#안녕 on Music</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    
	  <atom:link href="https://music.devkuma.com/tags/#%EC%95%88%EB%85%95/index.xml" rel="self" type="application/rss+xml" />
    
    
      
        
      
    
    
    <item>
      <title></title>
      <link>https://music.devkuma.com/track/0023/509225b6/1a0dc7f7/</link>
      <pubDate>Sun, 29 Oct 2023 12:17:00 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/0023/509225b6/1a0dc7f7/</guid>
      <description>
        
        
        &lt;p&gt;작사: 이푸름, 유상균, 리트너, 강재수&lt;br&gt;
/ KY: &lt;a href=&#34;https://www.youtube.com/@TJKaraoke/search?query=82498&#34; target=&#34;_black&#34;&gt;82498&lt;/a&gt;&lt;/p&gt;
&lt;p&gt;너에게 다른 사람이&lt;br&gt;
생겼다는 소식을 듣게 되었어&lt;br&gt;
난 하루 종일 복잡한 마음이라서&lt;br&gt;
더 아무것도 할 수 없었어&lt;/p&gt;
&lt;p&gt;날 보며 웃는 너의 행복했던 모습이&lt;br&gt;
이제는 아픔이 되고&lt;br&gt;
그렇게 니 생각은 자꾸만 깊어져&lt;br&gt;
추억 속 어딘가로 날 데려가는데&lt;/p&gt;
&lt;p&gt;이젠 너에게&lt;br&gt;
사랑을 말할 수 없고&lt;br&gt;
이별도 쉽게 못하고&lt;/p&gt;
&lt;p&gt;믿기지가 않아&lt;br&gt;
난 어떡해야 해&lt;/p&gt;
&lt;p&gt;우리의 사랑이 끝났다 해도&lt;br&gt;
다신 볼 수 없어도&lt;br&gt;
하나만 기억해 줘&lt;br&gt;
함께 한 시절&lt;/p&gt;
&lt;p&gt;우리가 함께했던 모든 순간순간이&lt;br&gt;
이제는 돌아오지 않는데&lt;br&gt;
덩그러니 남겨진 너의 사진 보면&lt;br&gt;
추억 속 그곳에선 함께인 것만 같아&lt;/p&gt;
&lt;p&gt;이젠 너에게&lt;br&gt;
사랑을 말할 수 없고&lt;br&gt;
이별도 쉽게 못하고&lt;/p&gt;
&lt;p&gt;믿기지가 않아&lt;br&gt;
난 어떡해야 해&lt;/p&gt;
&lt;p&gt;우리의 사랑이 끝났다 해도&lt;br&gt;
다신 볼 수 없어도&lt;br&gt;
하나만 기억해 줘&lt;br&gt;
함께 한 시절&lt;/p&gt;
&lt;p&gt;다 끝인가 봐&lt;br&gt;
되돌릴 수 없는 걸&lt;br&gt;
난 알고 있지만&lt;/p&gt;
&lt;p&gt;너에게 사랑을 말할 수 없다&lt;/p&gt;
&lt;p&gt;내 맘은 아직 널 그리는데&lt;br&gt;
여전히 난 너에게&lt;br&gt;
스쳐버린 사람 그런 추억일까&lt;/p&gt;
&lt;p&gt;시간이 저 멀리 널 데려가도&lt;br&gt;
더는 볼 수 없어도&lt;br&gt;
하나만 기억해줘&lt;br&gt;
함께 한 시절&lt;/p&gt;

      </description>
      
      <category>/#안녕</category>
      
    </item>
    
    <item>
      <title></title>
      <link>https://music.devkuma.com/track/0023/509225b6/fafefeb5/</link>
      <pubDate>Sun, 29 Oct 2023 12:16:57 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/0023/509225b6/fafefeb5/</guid>
      <description>
        
        
        &lt;p&gt;작사: JUNE, CHRIS지훈, 유상균, 리트너&lt;br&gt;
/ KY: &lt;a href=&#34;https://www.youtube.com/@TJKaraoke/search?query=82894&#34; target=&#34;_black&#34;&gt;82894&lt;/a&gt;&lt;/p&gt;

&lt;div style=&#34;position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;&#34;&gt;
  &lt;iframe src=&#34;https://www.youtube.com/embed/SS5mdirbJKQ&#34; style=&#34;position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;&#34; allowfullscreen title=&#34;YouTube Video&#34;&gt;&lt;/iframe&gt;
&lt;/div&gt;

&lt;hr&gt;
&lt;p&gt;꿈에 너의 번홀&lt;br&gt;
연락처에 다시 올렸어&lt;br&gt;
이미 바뀐지 오랜데&lt;/p&gt;
&lt;p&gt;너의 번홀 등록하니&lt;br&gt;
떠오른 너의 얼굴&lt;br&gt;
참 오랜만에 보았어&lt;/p&gt;
&lt;p&gt;모두 잠든 밤&lt;br&gt;
너에게 무슨 말을 건넬까&lt;br&gt;
어차피 꿈인데&lt;br&gt;
왜 넌 망설이니&lt;/p&gt;
&lt;p&gt;꿈에서 본 너에게&lt;br&gt;
말하다 꿈을 깰 까봐&lt;br&gt;
아무 말 못한 채 바라보고 있어&lt;/p&gt;
&lt;p&gt;몇 번의 이별을&lt;br&gt;
해야 괜찮니&lt;br&gt;
이번엔 안되니&lt;br&gt;
그만 아프고 싶은데&lt;/p&gt;
&lt;p&gt;눈물이 흐른다&lt;br&gt;
내 기억 추억 모두 흐른다 아~&lt;br&gt;
웃는다 이별이다&lt;br&gt;
안녕&lt;/p&gt;
&lt;p&gt;드디어 너에게&lt;br&gt;
도착한 설레는 메시지&lt;br&gt;
안녕 잘 지냈니&lt;br&gt;
왠지 아픈 그 말&lt;/p&gt;
&lt;p&gt;너에게 잘해주지 못한&lt;br&gt;
일만 떠올리다&lt;br&gt;
나도 모를 전활 네게 걸고 있어&lt;/p&gt;
&lt;p&gt;몇 번의 이별을&lt;br&gt;
해야 괜찮니&lt;br&gt;
이번엔 안되니&lt;br&gt;
그만 아프고 싶은데&lt;/p&gt;
&lt;p&gt;눈물이 흐른다&lt;br&gt;
내 기억 추억 모두 흐른다 아아-&lt;br&gt;
이제 그만 정말 그만&lt;/p&gt;
&lt;p&gt;안녕&lt;br&gt;
이렇게 널 여러 번 보냈어&lt;br&gt;
이제 그만 널 잊고 싶은데&lt;/p&gt;
&lt;p&gt;꿈에서도&lt;br&gt;
우린 이별하지 못하고&lt;br&gt;
이렇게 가슴 터지게 눈물만 흐른다&lt;/p&gt;
&lt;p&gt;몇 번의 이별을&lt;br&gt;
해도 난 너야&lt;br&gt;
다시 제자리야&lt;br&gt;
그저 내가 사는 이유&lt;/p&gt;
&lt;p&gt;너야&lt;br&gt;
아무리 이별&lt;br&gt;
요란하게 해봐도&lt;br&gt;
또 그 자리&lt;br&gt;
절대 비워지지 않는 네 자리&lt;/p&gt;
&lt;p&gt;(End)&lt;/p&gt;

      </description>
      
      <category>/#안녕</category>
      
    </item>
    
    <item>
      <title>먼데이키즈,#안녕 - 이 세상 끝까지</title>
      <link>https://music.devkuma.com/track/1106/ea7358e0/1e2dccab/</link>
      <pubDate>Sun, 22 Oct 2023 02:28:07 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/1106/ea7358e0/1e2dccab/</guid>
      <description>
        
        
        &lt;p&gt;작곡: 빅가이로빈 / 작사: 빅가이로빈&lt;br&gt;
KY: &lt;a href=&#34;https://www.youtube.com/@KARAOKEKY/search?query=93004&#34; target=&#34;_black&#34;&gt;93004&lt;/a&gt; / KY: &lt;a href=&#34;https://www.youtube.com/@TJKaraoke/search?query=83739&#34; target=&#34;_black&#34;&gt;83739&lt;/a&gt;&lt;/p&gt;
&lt;p&gt;어떤 말을 해 볼까&lt;br&gt;
생각보다 떨려서&lt;br&gt;
머뭇거리는 내가&lt;br&gt;
참 어색해&lt;br&gt;
서로를 잘 몰랐던&lt;br&gt;
우리가 여기&lt;br&gt;
두 손을 마주 잡고서&lt;br&gt;
평생을 약속하네요&lt;br&gt;
어느새 시간이 흘러서&lt;br&gt;
지금을 추억하는 날에&lt;br&gt;
우리는 어떤 미소들로&lt;br&gt;
바라보게 될까&lt;br&gt;
사랑하는&lt;br&gt;
그대여&lt;br&gt;
이젠 매일&lt;br&gt;
내 옆에 있어요&lt;br&gt;
분명하게 기억해&lt;br&gt;
우리의 처음을&lt;br&gt;
그대만을 따라서&lt;br&gt;
내가 할 수 있는&lt;br&gt;
작은 무언가 있기를&lt;br&gt;
영원할 우릴 위해&lt;br&gt;
이 세상&lt;br&gt;
끝까지&lt;br&gt;
거센 바람 휘몰아쳐도&lt;br&gt;
고단함에 얼굴 붉혀도&lt;br&gt;
그대 내 손 놓치지 말고&lt;br&gt;
함께해 줄래요&lt;br&gt;
사랑하는&lt;br&gt;
그대여&lt;br&gt;
이젠 매일&lt;br&gt;
내 옆에 있어요&lt;br&gt;
분명하게 기억해&lt;br&gt;
우리의 처음을&lt;br&gt;
그대만을 따라서&lt;br&gt;
내가 할 수 있는&lt;br&gt;
작은 무언가 있기를&lt;br&gt;
영원할 우릴 위해&lt;br&gt;
이 세상&lt;br&gt;
끝까지&lt;br&gt;
험난한 세상 속에&lt;br&gt;
내가 버틸 수 있던 건&lt;br&gt;
나를 바라봐 주는&lt;br&gt;
그대 때문에&lt;br&gt;
혼자여서 외롭던&lt;br&gt;
그랬던 난 이제 없어&lt;br&gt;
내 마음에&lt;br&gt;
그대가 가득 차 있으니&lt;br&gt;
그대&lt;br&gt;
내게 와 줘서&lt;br&gt;
더는 아무것도&lt;br&gt;
두렵지 않게 된걸요&lt;br&gt;
이제는 내가 그댈&lt;br&gt;
채워 주려 해요&lt;/p&gt;

      </description>
      
      <category>먼데이키즈</category>
      
      <category>/#안녕</category>
      
    </item>
    
    <item>
      <title>먼데이 키즈(Monday Kiz),#안녕 - 이 세상 끝까지</title>
      <link>https://music.devkuma.com/track/1106/17d585b9/1e2dccab/</link>
      <pubDate>Sun, 15 Oct 2023 23:58:05 +0900</pubDate>
      
      <guid>https://music.devkuma.com/track/1106/17d585b9/1e2dccab/</guid>
      <description>
        
        
        
&lt;div style=&#34;position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;&#34;&gt;
  &lt;iframe src=&#34;https://www.youtube.com/embed/qrBMbvNII9s&#34; style=&#34;position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;&#34; allowfullscreen title=&#34;YouTube Video&#34;&gt;&lt;/iframe&gt;
&lt;/div&gt;

&lt;hr&gt;
&lt;p&gt;어떤 말을 해볼까&lt;br&gt;
생각보다 떨려서&lt;br&gt;
머뭇거리는 내가 참 어색해&lt;/p&gt;
&lt;p&gt;서로를 잘 몰랐던&lt;br&gt;
우리가 여기&lt;br&gt;
두 손을 마주 잡고서&lt;br&gt;
평생을 약속하네요&lt;/p&gt;
&lt;p&gt;어느새 시간이 흘러서&lt;br&gt;
지금을 추억하는 날에&lt;br&gt;
우리는 어떤 미소들로&lt;br&gt;
바라보게 될까&lt;/p&gt;
&lt;p&gt;사랑하는 그대여&lt;br&gt;
이젠 매일 내 옆에 있어요&lt;br&gt;
분명하게 기억해&lt;br&gt;
우리의 처음을&lt;/p&gt;
&lt;p&gt;그대만을 따라서&lt;br&gt;
내가 할 수 있는&lt;br&gt;
작은 무언가 있기를&lt;br&gt;
영원할 우릴 위해&lt;br&gt;
이 세상 끝까지&lt;/p&gt;
&lt;p&gt;거센 바람 휘몰아쳐도&lt;br&gt;
고단함에 얼굴 붉혀도&lt;br&gt;
그대 내 손 놓치지 말고&lt;br&gt;
함께해 줄래요&lt;/p&gt;
&lt;p&gt;사랑하는 그대여&lt;br&gt;
이젠 매일 내 옆에 있어요&lt;br&gt;
분명하게 기억해&lt;br&gt;
우리의 처음을&lt;/p&gt;
&lt;p&gt;그대만을 따라서&lt;br&gt;
내가 할 수 있는&lt;br&gt;
작은 무언가 있기를&lt;br&gt;
영원할 우릴 위해&lt;br&gt;
이 세상 끝까지&lt;/p&gt;
&lt;p&gt;험난한 세상 속에&lt;br&gt;
내가 버틸 수 있던 건&lt;br&gt;
나를 바라봐 주는&lt;br&gt;
그대 때문에&lt;/p&gt;
&lt;p&gt;혼자여서 외롭던&lt;br&gt;
그랬던 난 이제 없어&lt;br&gt;
내 마음에 그대가&lt;br&gt;
가득 차 있으니&lt;/p&gt;
&lt;p&gt;그대 내게 와줘서&lt;br&gt;
더는 아무것도&lt;br&gt;
두렵지 않게 된 걸요&lt;br&gt;
이제는 내가 그댈&lt;br&gt;
채워주려 해요&lt;/p&gt;

      </description>
      
      <category>먼데이 키즈(Monday Kiz)</category>
      
      <category>/#안녕</category>
      
    </item>
    
  </channel>
</rss>
