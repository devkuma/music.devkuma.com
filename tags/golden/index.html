<!DOCTYPE html>
<html lang="en" itemscope itemtype="http://schema.org/WebPage"><head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />

  <link rel="icon" href="/favicon.svg">

  <title>
  Golden - Music
  </title>
  <meta name="description" content="" /><meta name="generator" content="Hugo 0.119.0"><link
    rel="stylesheet"
    href="https://music.devkuma.com/css/styles.min.970ec3263307ceba531b0b2a1f74e1544f092e20bc0c1366f9ddd1038a92d932.css"
    integrity="sha256-lw7DJjMHzrpTGwsqH3ThVE8JLiC8DBNm+d3RA4qS2TI="
  />
  
  

  <link rel="alternate" type="application/rss+xml" href="https://music.devkuma.com/tags/golden/index.xml" title="Music" />
  <meta property="og:title" content="Golden" />
<meta property="og:description" content="" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://music.devkuma.com/tags/golden/" />

  <meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Golden"/>
<meta name="twitter:description" content=""/>

  <meta itemprop="name" content="Golden">
<meta itemprop="description" content="">

  


<script async src="https://www.googletagmanager.com/gtag/js?id=G-7VSJLL8230"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', '{{ . }}');
</script>


</head>
<body class="dark:bg-gray-800 dark:text-white relative flex flex-col min-h-screen"><header class="container flex justify-between md:justify-between gap-4 flex-wrap p-6 mx-auto relative">
  <a href="https://music.devkuma.com/" class="capitalize font-extrabold text-2xl">
    
    <img src="/logo/256x256.png" alt="Music" class="h-8 max-w-full" />
    
  </a>
  <button class="mobile-menu-button md:hidden">
    <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
      <path stroke="none" d="M0 0h24v24H0z" fill="none"/>
      <line x1="4" y1="8" x2="20" y2="8" />
      <line x1="4" y1="16" x2="20" y2="16" />
    </svg>
  </button>
  <ul class="mobile-menu absolute z-10 px-6 pb-6 md:p-0 top-full left-0 w-full md:w-auto md:relative hidden md:flex flex-col md:flex-row items-end md:items-center gap-4 lg:gap-6 bg-white dark:bg-gray-800">

    
    <li><a href="/track">Songs</a></li>
    
    <li><a href="/tags">Tags</a></li>
    

    

    
    <li class="grid place-items-center">
      <span class="open-search inline-block cursor-pointer">
        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24" stroke-width="1.5"
          stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
          <path stroke="none" d="M0 0h24v24H0z" fill="none" />
          <circle cx="10" cy="10" r="7" />
          <line x1="21" y1="21" x2="15" y2="15" />
        </svg>
      </span>
    </li>
    

    
    <li class="grid place-items-center">
      <span class="toggle-dark-mode inline-block cursor-pointer">
        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" stroke-width="1.5"
          stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
          <path stroke="none" d="M0 0h24v24H0z" fill="none" />
          <circle cx="12" cy="12" r="3" />
          <line x1="12" y1="5" x2="12" y2="5.01" />
          <line x1="17" y1="7" x2="17" y2="7.01" />
          <line x1="19" y1="12" x2="19" y2="12.01" />
          <line x1="17" y1="17" x2="17" y2="17.01" />
          <line x1="12" y1="19" x2="12" y2="19.01" />
          <line x1="7" y1="17" x2="7" y2="17.01" />
          <line x1="5" y1="12" x2="5" y2="12.01" />
          <line x1="7" y1="7" x2="7" y2="7.01" />
        </svg>
      </span>
    </li>
    
  </ul>
</header>
<main class="flex-1">

  <h1 class="container px-6 pt-6 mx-auto text-3xl font-bold">Golden</h1>

  
  
  <div class="container p-6 mx-auto grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 lg:gap-8">
    <div class="p-2">
<a href="https://music.devkuma.com/track/0047/34538311/e0c0aeb6/">
  
  <div class="my-2 text-xl font-semibold">Golden - Hate Everything (Korean Ver.)</div>
  <div>
      <a href="/track/0047/34538311/e0c0aeb6/">
          <p class="summary">
              
                그냥 행복해 보이는데 내 머릿속의 사진 속엔 가장 예쁜 dress 입고 웃고 있는 니 곁의 나 비가 슬프게 내릴 때면 니가 싫어했던 그 영화처럼 회상만 가득한 날 모든 게 싫은 그런 날 이 노래를 부르는 것도 강한 척하는 것도 너 떠난 것도 내 단점들도 너 새로 만난 사랑도 hate everything Just hate everything right now 아무리 줘도 부족한걸 알면서도 난 그렇게 널 영원히 옆에 두려고 아파하는 날 버려두고 비가 슬프게 내릴 때면 니가 싫어했던 그 영화처럼 후회만 가득한 날 모든 게 싫은 그런 날 이 노래를 부르는 것도 강한 척하는 것도 너 떠난 것도 내 단점들도 니 새로 만난 사랑도 hate everything Just hate everything right now 되돌릴 수 있다면 다 지워 버릴 수 있다면 아름다웠던 시간들 Now its just killing me oh baby 이 노래를 부르는 것도 강한 척하는 것도 너 떠난 것도 내 단점들도 너 새로 만난 사랑도 hate everything Just hate everything right now Hate everything Just hate everything right now
              
          </p>
      </a>
  </div>
</a>

  

</div>
<div class="p-2">
<a href="https://music.devkuma.com/track/1107/b947c4b7/5b4d76fe/">
  
  <div class="my-2 text-xl font-semibold">박재범,Golden,pH-1 - Afternoon</div>
  <div>
      <a href="/track/1107/b947c4b7/5b4d76fe/">
          <p class="summary">
              
                Baby 눈을 감아 이제 더는 없어 드라마 어딜 가도 너만 찾아 손을 잡고 멀리 떠나자 Ima love you in the morning And in the evening And even in the afternoon In the morning and in the evening And even in the afternoon 언제부턴지 참지 못하고 이렇게 나 천천히 빠져 가는 건지 침착하려고 잘 보이려고 노력하는 내 모습 어떨지 몰라도 설명할 수 없는 감정들뿐인데 난 If you ever leave 모든 걸 잃은 듯 슬퍼할 내 모습이 oh no Baby 눈을 감아 이제 더는 없어 드라마 어딜 가도 너만 찾아 손을 잡고 멀리 떠나자 Ima love you in the morning And in the evening And even in the afternoon In the morning and in the evening And even in the afternoon Issa vibe 분위기 잡아 봐 girl Its alright 고민을 내려놔 oh 나는 바라는 게 없어 니 마음뿐 Yeah my love is real 간절히 기도해 Can you love me too 난 니가 필요해 널 떠날 일은 없어 my girl 걱정 마 설명할 수 없는 감정들뿐인데 난 If you ever leave 모든 걸 잃은 듯 슬퍼할 내 모습이 oh no Ima love you like Im Marley 지금 아주 멀리 떠나자 둘만을 위해서 준비된 party 딴생각은 트렁크 안에 drop top Got us whippin Thru the city like a cop car Got plenty of cash in the bag 하고픈 거 다 해 내 옆에 있을 때 모두 현실이 돼 Welcome to the lifestyle You been dreaming like forever 사진 대신 두 눈 속에 새겨 Baby 눈을 감아 이제 더는 없어 드라마 어딜 가도 너만 찾아 손을 잡고 멀리 떠나자 Ima love you in the morning And in the evening And even in the afternoon In the morning and in the evening And even in the afternoon
              
          </p>
      </a>
  </div>
</a>

  

</div>
<div class="p-2">
<a href="https://music.devkuma.com/track/0070/4506f92d/b2bf6869/">
  
  <div class="my-2 text-xl font-semibold">pH-1,Golden,서동현(BIG Naughty),박재범 - Oscar</div>
  <div>
      <a href="/track/0070/4506f92d/b2bf6869/">
          <p class="summary">
              
                Every good day start with a good yawn 잠을 푹 잤더니 시간은 두 시야 밖에 나갈 준비 하지 너를 만나러 너와 sipping 하는 커피는 너무 달아 넌 당부했지 오해하지 마 사귈 마음 없어 절대 고백하지 마 And it cut my heart open Put me in a coffin 그 말과 동시에 부서진 존심 침착 gotta relax Hollywood 웃음 짓네 Ha ha ha laugh like its funny 오스카를 원해 아카데미 원해 긴장 안 한 듯이 영화배우처럼 제스처는 큼직 Ha ha ha laugh like its funny 오스카를 원해 아카데미 원해 Ill be the one 니가 원하는 모든 역 다 완벽하게 소화해 낼게 Ill be the one 아무한테 안 보여 준 내 모습 다 너 원하는 건 Ill be the one Oh O O 잠깐만이여 Oh no O 잠깐 wait me up 어 아 음 길을 물으려 하는 건 아냐 사실 할 말이 있어서 오 오 오늘 시간 있어요 Uh oh 없어도 내게 와 줘요 일이 많아도 오 잠깐만 욜로 와 ay Okay 너를 데리러 갈게 너는 어디야 Oh baby 그냥 call me up Olleh 난 니 man이여 엄마 다음으로 너를 사랑해 girl 나한테 와 수많은 생각들이 내 머리를 스칠 때마다 드는 한 가지 비밀은 나는 바로 아마도 아 맞다 사실 그게 음 I think I love you so much Ill be the one 니가 원하는 모든 역 다 완벽하게 소화해 낼게 Ill be the one 아무한테 안 보여 준 내 모습 다 너 원하는 건 Ill be the one Ill be the one To give you the best of me Baby girl just leave it up to me Yeah you know that Im the one to be yeah Ill be the one To give you the best of me Baby girl just leave it up to me Yeah you know that Im the one to be yeah 네게 필요한 역이 돼 줄게 Whenever you need someone 예상치 못할 큰 감동을 줄게 Whenever you need someone Ill be the one 니가 원하는 모든 역 다 완벽하게 소화해 낼게 Ill be the one 아무한테 안 보여 준 내 모습 다 너 원하는 건 Ill be the one
              
          </p>
      </a>
  </div>
</a>

  

</div>

  </div><div class="grid place-items-center mb-8">
  
</div>
<div class="bg-blue-100 dark:bg-gray-900">
  <div class="container px-4 py-12 mx-auto max-w-4xl grid grid-cols-1 md:grid-cols-2 gap-4 items-center">
    <div>
      <div class="text-2xl font-bold mb-2"></div>
      <p class="opacity-60"></p>
    </div>

    <ul class="flex justify-center gap-x-3 flex-wrap gap-y-2">
      
    </ul>
  </div>
</div>

    </main><footer class="container p-6 mx-auto flex justify-between items-center">
  <span class="text-sm font-light">
    
    Copyright © 2023 - The devkuma All Rights Reserved · All rights reserved
    
  </span>
  <span onclick="window.scrollTo({top: 0, behavior: 'smooth'})" class="p-1 cursor-pointer">
    <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" viewBox="0 0 24 24" stroke-width="1.5"
      stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
      <path stroke="none" d="M0 0h24v24H0z" fill="none" />
      <path d="M18 15l-6 -6l-6 6h12" />
    </svg>
  </span>
</footer>

<div class="search-ui absolute top-0 left-0 w-full h-full bg-white dark:bg-gray-800 hidden">
  <div class="container max-w-3xl mx-auto p-12">
    <div class="relative">
      <div class="my-4 text-center text-2xl font-bold">Search</div>

      <span class="p-2 absolute right-0 top-0 cursor-pointer close-search">
        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" stroke-width="1.5"
          stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round">
          <path stroke="none" d="M0 0h24v24H0z" fill="none" />
          <line x1="18" y1="6" x2="6" y2="18" />
          <line x1="6" y1="6" x2="18" y2="18" />
        </svg>
      </span>
    </div>

    <input type="search" class="py-2 px-3 w-full dark:text-black border dark:border-transparent"
      placeholder="Enter search query" />

    <div class="search-results text-lg font-medium my-4 hidden">Results</div>
    <ul class="search-list my-2">

    </ul>

    <div class="no-results text-center my-8 hidden">
      <div class="text-xl font-semibold mb-2">No results found</div>
      <p class="font-light text-sm">Try adjusting your search query</p>
    </div>
  </div>
</div>





<script src="https://music.devkuma.com/js/scripts.min.js"></script>


<script async src="https://www.googletagmanager.com/gtag/js?id=G-7VSJLL8230"></script>
<script>
var doNotTrack = false;
if (!doNotTrack) {
	window.dataLayer = window.dataLayer || [];
	function gtag(){dataLayer.push(arguments);}
	gtag('js', new Date());
	gtag('config', 'G-7VSJLL8230', { 'anonymize_ip': false });
}
</script>





<script>
  
  const darkmode = document.querySelector('.toggle-dark-mode');
  function toggleDarkMode() {
    if (document.documentElement.classList.contains('dark')) {
      document.documentElement.classList.remove('dark')
      localStorage.setItem('darkmode', 'light')
    } else {
      document.documentElement.classList.add('dark')
      localStorage.setItem('darkmode', 'dark')
    }
  }
  if (darkmode) {
    darkmode.addEventListener('click', toggleDarkMode);
  }

  const darkStorage = localStorage.getItem('darkmode');
  const isBrowserDark = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;

  if (!darkStorage && isBrowserDark) {
    document.documentElement.classList.add('dark');
  }

  if (darkStorage && darkStorage === 'dark') {
    toggleDarkMode();
  }
</script>


<script>
  const mobileMenuButton = document.querySelector('.mobile-menu-button')
  const mobileMenu = document.querySelector('.mobile-menu')
  function toggleMenu() {
    mobileMenu.classList.toggle('hidden');
    mobileMenu.classList.toggle('flex');
  }
  if(mobileMenu && mobileMenuButton){
    mobileMenuButton.addEventListener('click', toggleMenu)
  }
</script>
</body>
</html>
